### DISCUSSION TITLE
Cannot add the config.json

### DISCUSSION THREAD

**Discussion by quamnana** [acheong08/ChatGPT#145] [Q&A]
I am trying to use this package in CLI mode but I get the message below:
Please create and populate config.json to continue

Please can someone help me with how to get the configuration done?
Thanks

**Comment by acheong08** [comment]
@Kobby08 https://github.com/acheong08/ChatGPT/wiki/Setup

**Reply by quamnana** [reply]
I have cloned the repo and added the required values in the config.json.  Then I ran the CLI command but it doesn't work.

Am I doing something wrong?
Can you help me with an elaborated explanation on how to properly set it up. I am using a macbook m1
Thanks

**Reply by acheong08** [reply]
You are probably in the wrong current working directory. `cd <path/to/repo>` and try again

**Reply by quamnana** [reply]
Thanks. It is working now. Great job with this project ✌️ ❤️
